project/src/const.ts

```diff
@@ -0,0 +1,14 @@
+enum AppRoute {
+  LOGIN = '/login',
```

Comment:
```suggestion
  Login = '/login',
```
На этом курсе другие правила)